openvdb/Grid.h

```diff
@@ -501,6 +510,11 @@ class OPENVDB_API GridBase: public MetaMap
     /// @brief Initialize with an identity linear transform.
     GridBase(): mTransform(math::Transform::createLinearTransform()) {}
 
+#if OPENVDB_ABI_VERSION_NUMBER >= 7
+    /// @brief Initialize with metadata and a transform.
+    GridBase(math::Transform::Ptr xform, const MetaMap& meta): MetaMap(meta), mTransform(xform) { }
```

Comment:
This needs to throw (see `GridBase::setTransform`) if the transform pointer is null.